tutorials/intro-to-geospatial-analysis/index.md

```diff
@@ -0,0 +1,449 @@
+---
+title: An Introduction to Geospatial Analysis
+description: This workshop provides a fundamental overview of remote sensing and geospatial analysis using the JavaScript API of Earth Engine.
+author: TC25
+tags: introductory, GIS, remote sensing, Google Earth Engine
+date_published: 2019-10-09
+---
+
+[Geometries - Open In Code Editor](https://code.earthengine.google.com/9d0bd55b88c3588dd189c3b512378a0b)
+
+[Features - Open In Code Editor](https://code.earthengine.google.com/bb1980ff4b6ddab0262de00956f89dcb)
+
+[Images - Open In Code Editor - Geometries](https://code.earthengine.google.com/234ba0ab11dd82112602de05c0bc3bd8)
```

Comment:
Please ensure code doesn't contain references to personal repos. If possible please create features inline, or refer to a dataset in the data catalog.

Reply:
This might be harder to do since the tutorial is based on the shapefile (with neighborhood divisions) of a city. Is there a common gee account where I can ingest that shapefile? Otherwise, I can probably subset a data from the US census tracts feature collection.

Reply:
I'd go with the census tracts.

Reply:
Done with counties.